src/app/services/ui/ui.interaction.service.ts

```diff
@@ -394,6 +394,10 @@ export class UiInteractionService implements OnDestroy {
     this.showPerlenketteSubject.next(false);
   }
 
+  closeTrainrunDialog() {
+    this.showTrainrunDialogSubject.next(null);
```

Comment:
Since we now send null here, could we update the definition of `showTrainrunDialogSubject` to `Subject<TrainrunDialogParameter | null>` to indicate that this subject now sometimes receives null?